.eslintrc.json

```
@@ -0,0 +1,13 @@
+{
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": "error"
+  },
+  "extends": [
+    "plugin:prettier/recommended",
+    "prettier/flowtype",
+    "prettier/react",
+    "prettier/standard"
+  ],
+  "prettier/prettier": ["error", {"singleQuote": true, "parser": "flow"}]
+}
```

Вот правильный `.eslintrc.json`:

```suggestion
{
  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": "error"
  },
  "extends": [
    "plugin:prettier/recommended",
    "prettier/flowtype",
    "prettier/react",
    "prettier/standard"
  ],
  "prettier/prettier": ["error", {"singleQuote": true, "parser": "flow"}]
}
{
  "env": {
    "es6": true,
    "browser": true,
    "commonjs": true
  },
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    },
    "ecmaVersion": 6,
    "sourceType": "module"
  },
  "plugins": ["react", "prettier"],
  "rules": {
    "prettier/prettier": ["error", { "singleQuote": false }]
  },
  "extends": [
    "htmlacademy/es6",
    "plugin:react/recommended",
    "plugin:prettier/recommended"
  ],
  "settings": {
    "react": {
      "version": "16"
    }
  }
}
```

---

И удали `eslintrc.yml`, иначе у тебя два конфига и твой линтер смотрит на `yml`-версию